index.html

```diff
@@ -71,17 +71,165 @@ <h3 class="title">Skills</h3>
       <div class="line"></div>
 
       <div class="part-three">
-          <ul  class="social">
-            <li><a href=""><i class="fa-brands fa-github-square"></i></a></li>
-            <li><a href=""><i class="fa-brands fa-linkedin-in"></i></a></li>
-            <li><a href=""><i class="fa-regular fa-hand-peace"></i></a></li>
-            <li><a href=""><i class="fa-brands fa-twitter"></i></a></li>
-            <li><a href=""><i class="fa-solid fa-m"></i></a></li>
-          </ul>
+        <ul class="social">
+          <li>
+            <a href=""><i class="fa-brands fa-github-square"></i></a>
+          </li>
+          <li>
+            <a href=""><i class="fa-brands fa-linkedin-in"></i></a>
+          </li>
+          <li>
+            <a href=""><i class="fa-regular fa-hand-peace"></i></a>
+          </li>
+          <li>
+            <a href=""><i class="fa-brands fa-twitter"></i></a>
+          </li>
+          <li>
+            <a href=""><i class="fa-solid fa-m"></i></a>
+          </li>
+        </ul>
         <div class="work">
           <div><i class="fa-solid fa-arrow-down"></i><span> Work</span></div>
         </div>
       </div>
     </section>
+
+    <!-- Middle part -->
+    <section class="middle-section">
+      <h3 class="section-title work-title">
+        MY RECENT <br />
+        WORK
+      </h3>
+      <div class="line"></div>
+      <div class="work-content">
+        <!-- Work card one -->
+        <div class="work-card">
+          <img src="images/work_img.png" alt="" />
+          <div class="card">
+            <h4>Multi-Post Stories</h4>
+            <div class="section-text card-text">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </div>
+            <div class="cart-link">
+              <ul>
+                <li><a href="">css</a></li>
+                <li><a href="">html</a></li>
+                <li><a href="">bootstrap</a></li>
+                <li><a href="">Ruby</a></li>
+              </ul>
+            </div>
+            <button type="button" class="btn">
+              <span>See Project</span>
+            </button>
```

Comment:
- [ ] Please The button  "See project"  follows the button styling: Active, Hover, and Pressed 
- [ ] see design: [templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10)


![kk](https://user-images.githubusercontent.com/63972670/164575552-08162817-89b6-47d2-8117-e33044cadf95.PNG)